client/js/screens/Profile/ProfileContainer.js

```diff
@@ -8,6 +11,44 @@ export default class ProfileContainer extends Component {
   }
 
   render() {
-    return <Profile />;
+    return (
+      <Query
+        query={USER_QUERY}
+        variables={{ id: "cjx2bg22z08h90166zqgy3ium" }}
+        fetchPolicy="network-only"
+      >
+        {({ loading, error, data }) => {
+          console.log("data", data);
+          if (loading) return <Text>Loading...</Text>; // replace by Loader component
+          if (error) return <p>{`Error! ${error.message}`}</p>;
```

Comment:
React Native does not have a `p` tag. Please use react native components.

Reply:
Done.